app.js

```diff
@@ -1,31 +1,43 @@
 const express = require("express");
 const app = express();
-const endpoints = require("./endpoints.json")
+const endpoints = require("./endpoints.json");
 
 app.use(express.json());
```

Comment:
you're not using this line yet, it's used for parsing incoming bodies for example in a post request